charts/k8s-integration/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: k8s-integration
+description: A Helm chart for miggo's k8s-integration
+type: application
+version: 0.0.1
+appVersion: "0.0.1"
+dependencies:
+  - name: k8s-read
+    repository: https://miggo-io.github.io/k8s-read
+    version: 0.0.1
+    condition: k8s-read.enabled
+  - name: static-sbom
+    repository: https://miggo-io.github.io/miggo-helm-charts
```

Comment:
i dont get the repo paths, why this is referring miggo-helm-charts and the previous one to k8s-read?

Reply:
Actually this is the correct one and k8s-read is the mistake.

We have one repo with multiple charts, the `name` specify the chart while this property specify the repo